SCons/Script/Main.py

```diff
@@ -1356,12 +1356,6 @@ def order(dependencies):
     # various print_* settings, tree_printer list, etc.
     BuildTask.options = options
 
-    is_pypy = platform.python_implementation() == 'PyPy'
```

Comment:
why are you removing is_pypy ?

Reply:
It and `is_37_or_later` were only used in the commented-out `python_has_threads` variable. I could just leave it be, but I was already removing the redundant `is_37_or_later` which would otherwise just be an empty variable always returning `True`, so removing that whole vestigial block felt more appropriate